src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -184,6 +184,9 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
           --mdc-theme-surface: white;
           --mdc-list-vertical-padding: 5px;
           --mdc-list-side-padding: 25px;
```

Comment:
Once more, It's not necessary unless the attribute `FixedMenuPosition` is used in `mwc-select`.